packages/react-components/react-calendar-compat/src/components/CalendarDayGrid/useCalendarDayGridStyles.styles.ts

```diff
@@ -174,7 +174,6 @@ const useDaySingleSelectedStyles = makeStyles({
 const useWeekRowStyles = makeStyles({
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31596&commitId=a9d845f326fa4c3144634d4c7429b7d48159e52c&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask - RTL.normal.chromium.png|6| Changed|
|Avatar Converged.basic - RTL.normal.chromium.png|0| Added|
|Avatar Converged.basic.normal.chromium.png|0| Added|

</details>


  <details><summary><b>Card Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged.appearance - RTL.normal.chromium.png|0| Removed|

</details>


  <details><summary><b>Card Converged - Selectable</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged - Selectable.appearance focusable + selectable.click.chromium.png|0| Added|
|Card Converged - Selectable.appearance focusable + selectable.hover.chromium.png|0| Added|
|Card Converged - Selectable.appearance focusable + selectable.normal.chromium.png|0| Added|
|Card Converged - Selectable.appearance focusable + selectable.selected.chromium.png|0| Added|

</details>


  <details><summary><b>SwatchPicker Converged</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SwatchPicker Converged.Spacing.default.chromium.png|0| Added|
|SwatchPicker Converged.Shape.default.chromium.png|0| Removed|
|SwatchPicker Converged.spacing.default.chromium.png|0| Removed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>